assets/entity_configs/units/unit1.json

```diff
@@ -8,5 +8,6 @@
   "attackPower": 50,
   "defense": 25,
   "sightRadius": 2,
-  "speed": 1
+  "speed": 1,
+  "moveRange": 2
```

Comment:
Wydawało mi się, że speed to jest właśnie moveRange

Reply:
no w sumie, to można teraz usunąć speed, moveRange lepiej brzim